WcaOnRails/app/models/competition.rb

```diff
@@ -593,6 +610,10 @@ def country
     Country.find_by_id(countryId)
   end
 
+  def continent
+    country ? Continent.find_by_id(country.continentId) : nil
```

Comment:
That aside, it's weird to do this by hand. Something like this should work, and simplify things, no?

`belongs_to :country,  foreign_key: "countryId"`